doc/source/index.rst

```diff
@@ -34,13 +34,24 @@ The freud library uses single-precision `NumPy arrays <https://www.numpy.org/>`_
 
 .. toctree::
    :maxdepth: 2
-   :caption: Contents
+   :caption: Getting Started
 
    installation
    examples
+
+.. toctree::
+   :maxdepth: 2
+   :caption: API
+
    modules
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Reference
+
    development
-   citations
+   citing
+   references
```

Comment:
I think our references section is a little strange to be honest. I think we should put all citations inline (i.e. in the context of whatever method they're there for) and get rid of this section. We should also get more consistent with having citations, but that's a longer term things (maybe good for a hackathon to ask someone to identify features that need a citation).

Reply:
Discussed offline, we will do this in a separate PR.